doc/code_of_conduct_reporting_guidelines.rst

```diff
@@ -0,0 +1,59 @@
+.. _Zephyr_Code_of_Conduct_Reporting_Guidelines:
+
+Code of Conduct Reporting Guidelines
+####################################
+
+We are committed to providing members of our community with a safe and productive environment. If you experience or witness violations of the Zephyr Project Code of Conduct please use these reporting guidelines outlined by the Zephyr Projects’ Code of Conduct Working Group. It's used to provide us with as much relevant information as possible when we respond to an issue to make sure we're consistent and fair. If you have questions please contact us at **report@zephyrproject.org**
+
+If you believe someone is violating the Zephyr Code of Conduct (CoC) we ask that you report it to the Zephyr Project by emailing **report@zephyrproject.org**. All reports will be kept confidential. In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+**If you believe anyone is in physical danger, please notify appropriate law enforcement first.** If you are unsure what law enforcement agency is appropriate, please include this in your report.
+
+If you are unsure whether the incident is a violation, or whether the space where it happened is covered by this Code of Conduct, we encourage you to still report it. We would much rather have a few extra reports where we decide to take no action, rather than miss a report of an actual violation. We do not look negatively on you if we find the incident is not a violation. And knowing about incidents that are not violations, or happen outside our spaces, can also help us to improve the Code of Conduct or the processes surrounding it.
+
+Reporting
+*********
+
+In your report please include:
+ * Your contact info (so we can get in touch with you if we need to follow up).
```

Comment:
add a blank line before a list, and don't indent the list items:

    In your report please include:

    * Your contact info (so we can get in touch with you if we need
      to follow up).
    * Names (real, nicknames, or pseudonyms) of any individuals
      involved. If there were other witnesses besides you, please
      try to include them as well.